### DISCUSSION TITLE
How to go about developing the Wiki

### DISCUSSION THREAD

**Discussion by rexelbartolome** [freeplane/docs#15] [General]
## Focus
So I think firstly, just like what Dimitry said, making sure that our 'one source of truth' (which is our new wiki page in Github) is working, reliable, and is finished to a certain degree, then everything else can follow. Since we can jump off of the new wiki into other tasks such as linking it inside Freeplane, converting wiki pages into mindmaps, or making it into a tutorial addon as Edo has showed etc

So we don't necessarily need to update/omit the tutorial mindmap and other resources just yet, we can decide what to do with them after we're generally happy with the state of the new docsify wiki.

Now that we're focused on the new wiki, the next step is to organize it into a more structured tree layout. 

## The Wiki's
I'll call this one the migrated wiki https://macmarrum.github.io/freeplane-wiki/migrated-from-mediawiki/# 

and this one the main wiki https://freeplane.github.io/wiki/#/

so the migrated wiki of course needs a lot more organization and nesting compared to the main wiki. I think the main wiki only needs a bit more polish. Then after the organization, it's now much easier to tell how we can merge these two together, deciding on which parts to overwrite/omit and also identify parts that need updating and call for contributors like "hey this part needs to be rewritten, if you're familiar with this aspect of freeplane, feel free to contribute" much like, if I remember correctly, sifran mentioned

So macmarrum, kindly explain how we could help you with organizing this

## Tutorial-O-Matic (ToM)
Then by the time we've merged it, maybe Edo's done with developing the tutorial-o-matic and then he could maybe teach it to us (or to anyone who's interested) in a call with screenshare just like what we did in spatial. That way Edo won't need to create formal documentation regarding tutorial-o-matic before we can use it. Then figure out which parts of the wiki we want to 'translate' into a ToM tutorial that's available inside Freeplane. 

It's probably only a handful of wiki pages going to be 'translated' e.g., quick start guide, formatting, and other beginner stuff. The rest can stay inside the wiki. For example, other mindmap software only has an onboarding tutorial that shows you how to make a child node, sibling node, and how to change the design of the nodes. It doesn't explain anything else.

## Tutorial Mindmap inside Freeplane
Since we're on the topic, I'd prefer if the ToM could replace the tutorial mindmap that comes with Freeplane. I never really used the tutorial mindmap myself, the "basic functions" mindmap is a bit too busy in my eyes to understand intuitively. 

And presenting the tutorial as a mind map kind of introduces an ouroboros problem: 

to read the tutorial mindmap, the user first needs to understand how to navigate Freeplane. but to understand how to navigate freeplane, the user needs to read the tutorial mindmap, but to read the mindmap... etc. 

Besides that, ToM has a linear structure while a mindmap can be treated in a non-linear fashion. If the user deletes/reorganizes the nodes into a different layout by accident, the user might have trouble navigating through it, while in ToM, the user can just go back to a previous 'slide' or 'step' and they can read the same information again and it wouldn't be possible for the user to accidentally delete a 'slide' because of how ToM behaves.

But of course, these are all just my thoughts on how we could go about it, tell me if you have another direction in mind.

@ofossum @macmarrum @EdoFro and I'm not sure who else, so feel free to tag others here. I don't know sifran's and 1d-box's tag over here in Github

**Comment by macmarrum** [comment]
The migrated wiki will be available at https://freeplane.github.io/wiki/migrated-from-mediawiki/

The migration is well in progress, though I've noticed some content that did not get converted properly – I will work on it. I plan to publish the migrated wiki soon. There'll still be portions of it that are only available in on the MediaWiki server – mostly mind maps and images. My idea is to leave the old wiki running, but under a different address, so that the content can still be accessed and/or migrated.

**Comment by sifran-github** [comment]
I sent a pull request and posted a discussion: https://github.com/freeplane/wiki/discussions/21
